### PR TITLE
test 'acf'  package

### PR DESCRIPTION
**Pull request by headupinclouds**
UPDATE FOR https://github.com/ruslo/hunter/pull/1399

* I have checked that this pull request contains only
  `.travis.yml`/`appveyor.yml` changes. All other changes send
  to https://github.com/ruslo/hunter. **[Yes]**

* I have checked that no toolchains removed from CI configs, they are commented
  out instead so other developers can enable them back easily and to simplify
  merge conflict resolution. **[Yes]**

* I have checked that for every commented out toolchain there is a link to the
  broken CI build page or to the minimum compiler requirements documentation
  so other developers can figure out what was the problem exactly. **[Yes]**

* I have checked that for every enabled toolchain corresponding package passed
  all stages of update cycle: test/merge/upload/release. **[Yes]**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -71,24 +69,13 @@ matrix:
...
     #   osx_image: xcode9.2
-    #   env: PROJECT_DIR=examples/acf TOOLCHAIN=ios-nocodesign-11-2-dep-9-3
-
-    # }
-
-    # Extra {
-
-    - os: osx
-      osx_image: xcode9.2
-      env: PROJECT_DIR=examples/acf TOOLCHAIN=ios-nocodesign-11-2-dep-9-3-armv7
```
Comment (ruslo) [inline .travis.yml]: This one is working, no need to remove.

**Review by ruslo — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (headupinclouds) [reply .travis.yml]: Okay.  I removed it since it wasn't in pkg.template or pkg.opencv.  I can add it back.

Comment (headupinclouds) [reply .travis.yml]: I've added those in an `# Extra { ` `# }` block

**Comment by headupinclouds** [comment]
New iOS toolchains added here:

https://github.com/headupinclouds/hunter/blob/befd86c4a2fc60dd40f2ceb997f8d52f10d3c67e/.travis.yml#L84-L90

Tests are passing:

https://travis-ci.org/headupinclouds/hunter/builds/364629366

File: .travis.yml [hunk 2]
```diff
@@ -71,24 +69,25 @@ matrix:
...
 
     # }
 
-    # Extra {
+    # Extra { 
 
     - os: osx
       osx_image: xcode9.2
-      env: PROJECT_DIR=examples/acf TOOLCHAIN=ios-nocodesign-11-2-dep-9-3-armv7
+      env: PROJECT_DIR=examples/OpenCV TOOLCHAIN=ios-nocodesign-11-2-dep-9-3-armv7 VERBOSE=0
```
Comment (ruslo) [inline .travis.yml]: `s,OpenCV,acf,`

Comment (ruslo) [reply .travis.yml]: so effectively there is no real changes introduced by this patch (?)

**Review by ruslo — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 2 above)
Comment (headupinclouds) [reply .travis.yml]: I grabbed the extra toolchains from `pkg.opencv` but missed the query-replace.  I've fixed this in the latest and applied the changes to `test.acf` as well.

Comment (ruslo) [reply .travis.yml]: I still see no real changes here like adding new toolchain.

Comment (headupinclouds) [reply .travis.yml]: I was missing your point.  I see now, a recent `pkg.template` was merged in yesterday here : https://github.com/ingenue/hunter/commit/bb365a67cf01b0f8e4a7dec84e14e4fda780a1bf#diff-354f30a63fb0907d4ad57269548329e3

I applied the same changes for `test.acf` tests and sent them back here without realizing they were already applied.

**Pull request closed without merging**